### ISSUE TITLE
Idea: dedicated website for the docs.

### ISSUE THREAD

**Issue by jelleroorda** [laravel/cashier-mollie#262]
I think it would be a good idea to have a dedicated website for the docs, as to make it easier to navigate it. The docs are quite substantial, so in my opinion it would be a lot easier to find what you are searching for (instead of crawling through the readme).

Any thoughts on this? (I understand this is most likely not a priority though)

**Comment by sandervanhooft** [comment]
I agree. We're looking into this.

**Comment by glamorous** [comment]
I saw there were dedicated docs on the Laravel website for Stripe and Paddle. Why not for Mollie also, it's a located at the Laravel Github account, so documentations should be available there too?

**Comment by sandervanhooft** [comment]
I've recently had a discussion about this with Taylor. As Cashier Mollie is not built by the core Laravel dev team, we'll have to find another place for the docs. For me it would also really hard to maintain the docs there without full control over the PR/build process.

We're looking into setting up a simple VuePress website now, which may allow us to include a bit more detailed and rich code samples.

**Comment by glamorous** [comment]
Sad to hear. Especially because the repository is inside the Laravel project I would suspect that it's on the mainsite or there's a link to the repo. I only found this repository through the Mollie website because I didn't see on the official docs that there was a package for it. I thought first it was by Mollie or 3rd party developer but then I saw it was on the Laravel project. Therefore the confusion about the docs.
